chore: internal docs for beta (#187)
storyblok/storyblok-cli: src/commands/migrations/run/README.md

# Migrations Run Command

The `migrations run` command allows you to execute migrations on stories in your Storyblok space.

> [!NOTE]

> > Before running migrations, you need to create them first using the `migrations generate` command. See [migrations generate](./generate/README.md) for more details.

Reviewer: ```suggestion
> Before running migrations, you must create them by running the `migrations generate` command. See [migrations generate](./generate/README.md) for more details.
```

> [!NOTE]

> > When running a migration, a snapshot of the story's content will be automatically created in the `.storyblok/migrations/{spaceId}/rollbacks` directory with a timestamp. These snapshots can be used to rollback changes if needed.

Reviewer: ```suggestion
> When running a migration, a snapshot of the story's content will be automatically created in the `.storyblok/migrations/{spaceId}/rollbacks` directory with a timestamp. These can be used to rollback changes if needed.
```
